packages/nuka/src/carousel.test.tsx

```diff
@@ -140,39 +158,45 @@ describe('Carousel', () => {
       pause: [32],
     };
     const slideCount = 8;
-    renderCarousel({
+    const { container } = renderCarousel({
       enableKeyboardControls: true,
       keyCodeConfig,
       slideCount,
       beforeSlide,
+      frameAriaLabel: 'keyboard',
+      landmark: true,
+      carouselId,
     });
 
-    const carouselFrame = screen.getByRole('region');
+    const sliderFrame = screen.getByTestId(`${carouselId}-slider-frame`);
```

Comment:
`frameAriaLabel` now references the whole carousel, this ensures the same slider frame element is being selected.